StreamChat.xcodeproj/project.pbxproj

```diff
@@ -8309,6 +8310,24 @@
 			shellPath = /bin/sh;
 			shellScript = "if which mint >/dev/null && mint which swiftgen; then\n  xcrun --sdk macosx mint run swiftgen config run --config ./Sources/StreamChatUI/.swiftgen.yml\nelse\n  echo \"Warning: Bootstrap not run, please run ./bootstrap.sh\"\nfi\n";
 		};
+		E3FBB83E27F737F10067608A /* SwiftLint */ = {
+			isa = PBXShellScriptBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			inputFileListPaths = (
+			);
+			inputPaths = (
+			);
+			name = SwiftLint;
+			outputFileListPaths = (
+			);
+			outputPaths = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = /bin/sh;
+			shellScript = "export PATH=\"$PATH:/opt/homebrew/bin\"\nif which swiftlint > /dev/null; then\n  swiftlint\nelse\n  echo \"warning: SwiftLint not installed, download from https://github.com/realm/SwiftLint\"\nfi\n";
+		};
```

Comment:
Having the phase in DemoApp means that when compiling just LLC or UI rule violations won't be highlighted. It's fine for now since we don't have another place where linting for both SDKs can happen but let's keep an eye on it and maybe have a tech discussion point. Linting SDKs separately is what we might finally end up with.

Reply:
Yeah, I agree on this @evsaev and it's a very good point